perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -244,6 +269,19 @@ def CleanupBucket(storage_schema):
     objects_to_cleanup = _ListObjects(storage_schema, FLAGS.bucket, prefix=None)
 
 
+def GenerateWritePayload(size):
+  payload_bytes = bytearray(size)
+  # don't use a for...range(100M), range(100M) will create a list of 100mm items
+  # with each item being the default object size of 30 bytes or so, it will lead
+  # to out of memory error.
+  for i in xrange(size):
+    payload_bytes[i] = ord(random.choice(string.letters))
+
+  payload_string = payload_bytes.decode('ascii')
+
+  return payload_bytes, payload_string
```

Comment:
I am fine with returning both payload_bytes and payload_string as is in this PR. (it's the same behavior today).

Good to know that you are working on a separate PR that will refactor this to 3 "backends", but that is probably not required for this PR to work. For now, I'd stay focused on getting this one and the object size distribution done first.